src/pages/ChangeLogPage.js

```diff
@@ -0,0 +1,132 @@
+import { makeStyles, Typography } from "@material-ui/core";
+import { format, isAfter, parseISO } from "date-fns";
+import React, { useEffect, useState } from "react";
+import { connect } from "react-redux";
+import { addSnackbar } from "../actions/notificationActions";
+import { fetchReleases } from "../services/changeLogService";
+import { useMobile } from "../utils/media-queries";
+import ReactMarkdown from "react-markdown";
+import remarkGfm from "remark-gfm";
+import PropTypes from "prop-types";
+
+const useStyles = makeStyles((theme) => ({
+    content: ({ isMobile }) => ({
+        padding: isMobile ? theme.spacing(0, 1) : theme.spacing(5),
+    }),
+    gray: {
+        color: "rgba(0, 0, 0, 0.54)",
+    },
+    releaseContainer: {
+        marginBottom: theme.spacing(4),
+    },
+    releaseHeader: {
+        display: "flex",
+        flexDirection: "row",
+        justifyContent: "space-between",
+        alignItems: "center",
+    },
+    releaseDate: {
+        marginRight: theme.spacing(2),
+        color: theme.palette.primary.dark,
+    },
+    releaseContent: {
+        padding: theme.spacing(2),
+    },
+}));
+
+export const ChangeLogPage = ({ addSnackbar }) => {
+    const isMobile = useMobile();
+    const classes = useStyles({ isMobile: isMobile });
+
+    const [releases, setReleases] = useState([]);
+    const [lastUpdateTS, setLastUpdateTS] = useState(null);
+
+    useEffect(() => {
+        const fetchData = async () => {
+            const releases = await fetchReleases().then((releases) => {
+                let lastTS = null;
+                const filteredReleases = releases.map((release, idx) => {
+                    const publishDate = parseISO(release.published_at);
+
+                    if (lastTS === null || isAfter(publishDate, lastTS))
+                        lastTS = publishDate;
+
+                    return {
+                        id: `release-${idx}`,
+                        name: release.name,
+                        date: format(publishDate, "dd-MM-yyyy"),
+                        body: release.body,
+                    };
+                });
+
+                if (lastTS !== null)
+                    setLastUpdateTS(format(lastTS, "dd-MM-yyyy"));
+                return filteredReleases;
+            });
+            setReleases(releases);
+        };
+
+        fetchData().catch((_) => {
```

Comment:
Is there a reason to use then/catch here instead of the `await` logic? Maybe it would be cleaner